app/modules/individuals/models.py

```diff
@@ -116,14 +116,26 @@ class Individual(db.Model, FeatherModel):
 
     featured_asset_guid = db.Column(db.GUID, default=None, nullable=True)
 
-    version = db.Column(db.BigInteger, default=None, nullable=True)
-
     encounters = db.relationship(
         'Encounter', back_populates='individual', order_by='Encounter.guid'
     )
 
     names = db.relationship('Name', back_populates='individual', order_by='Name.created')
 
+    # TODO check that these lengths are sensible
```

Comment:
1. comment should be _unrestricted_ (e.g. just `db.String()`)... we have LONG comments to migrate from wildbook
2. sex should _probably_ be validated.  but how?  maybe we could do for-real enum in db, but should confirm with team that is "ecologically unanimous" or something.   :grin: simpler might be to have a `set_sex()` we use instead.

fwiw, _my_ take on **the** acceptable values for sex: `null`, `unknown`, `male`, `female`.

Reply:
Comments now unrestricted (plus in sightings).
Sex still TOODO